src/items.rs

```diff
@@ -2294,7 +2320,7 @@ impl Rewrite for ast::Param {
                 let (before_comment, after_comment) =
                     get_missing_param_comments(context, self.pat.span, self.ty.span, shape);
                 result.push_str(&before_comment);
-                result.push_str(colon_spaces(context.config));
+                result.push_str(colon_spaces(context.config, false));
```

Comment:
Can you explain why we can pass `false` here? It would also be helpful if you could add an explanation to the other call sites where you've passed `false`.